deviceplugin/usb.go

```diff
@@ -78,6 +80,27 @@ func (dev *usbDevice) BusPath() (path string) {
 	return fmt.Sprintf(usbDevBus, dev.Bus, dev.BusDevice)
 }
 
+// readFileToUint16 reads the file at the given path, then returns a representation of that file as uint16.
+// Ignores newlines.
+// Returns an error if the file could not be read, or parsed as uint16.
+func readFileToUint16(path string) (out uint16, err error) {
+	bytes, err := os.ReadFile(path)
+	if err != nil {
+		// We can't read this file for some reason
+		return out, err
+	}
+	// To be safe, strip out newlines
+	datastr := strings.ReplaceAll(string(bytes), "\n", "")
+
+	// then attempt to parse as uint16.
+	dAsInt, err := strconv.ParseUint(datastr, 16, 16)
+	if err != nil {
+		return 0, fmt.Errorf("malformed device data %q: %w", datastr, err)
```

Comment:
above on line 90, we return `out, err` when there's an error, but here we explicitly return `0, err`. They should be equivalent since `out` will have its zero value, but we should pick one style and stick with it for consistency

Reply:
Concur, does 0d70ab4a look good?

Reply:
Looks great!